packages/cli/src/report/report-name-generator-builder.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { injectable } from 'inversify';
+import { padStart } from 'lodash';
+
+@injectable()
+export class ReportNameGeneratorBuilder {
+    public getDateSegment(scanDate: Date): string {
+        return `${scanDate.getFullYear()}${this.padStartWithZero(scanDate.getMonth() + 1, 2)}${this.padStartWithZero(
+            scanDate.getDate(),
+            2,
+        )}`;
+    }
+
+    public getTitleSegment(pageTitle: string): string {
```

Comment:
Is this for future use? It doesn't look like it's used anywhere at the moment

Reply:
Yeah, could be used later for the summary report name